src/wallet/wallet.cpp

```diff
@@ -1798,8 +1798,9 @@ CWallet::ScanResult CWallet::ScanForWalletTransactions(const uint256& start_bloc
     }
     double progress_current = progress_begin;
     while (block_height && !fAbortRescan && !chain().shutdownRequested()) {
+        m_scanning_progress = (progress_current - progress_begin) / (progress_end - progress_begin)
```

Comment:
Semicolon is still missing from this commit.

Reply:
Thanks, bad fixup :/